src/libdredd/src/mutation_replace_binary_operator.cc

```diff
@@ -63,9 +63,6 @@ std::string MutationReplaceBinaryOperator::GetExpr(
 
 bool MutationReplaceBinaryOperator::IsRedundantReplacementOperator(
     clang::BinaryOperatorKind op, clang::ASTContext& ast_context) const {
-  clang::Expr::EvalResult rhs_eval_result;
-  clang::Expr::EvalResult lhs_eval_result;
-
```

Comment:
I am curious as to why the CI passed last time - I would have expected a complaint about these not being used. Good spot this round.

Reply:
(This one doesn't need action so resolve it once digested. I'm just a bit mystified by why the compiler sometimes complains about unused variables but not always.)

Reply:
It is very confusing!